src/Common/Session/SessionUtil.php

```diff
@@ -185,10 +185,13 @@ public static function apiSessionStart($web_root): void
 
     public static function apiSessionCookieDestroy(): void
     {
-        (new SystemLogger())->debug(
-            "Destroying api session cookie",
-            ['name' => session_name(), 'cookieParams' => session_get_cookie_params()]
-        );
+        // mdsupport - No current session > No db connection > No log entry
+        if (class_exists('SystemLogger')) {
```

Comment:
I'm confused here.  Is the PSR4 autoloader not working here in this class?  How is there a class where the SystemLogger class won't exist?  Also, class_exist is going to look for a a \SystemLogger class, if the line is truly needed I believe you will need to use the following:
```
if (class_exists('OpenEMR\Common\Logging\SystemLogger')) {
```